### PR TITLE
Give training dataset resolution info in modelzoo

### PR DESCRIPTION
**Pull request by dreamerlin**

### CODE REVIEW

**Comment by dreamerlin** [comment]
@JoannaLXY @SuX97
Please fill the table together

**Comment by dreamerlin** [comment]
@JoannaLXY @SuX97
Please fill the table together

**Comment by dreamerlin** [comment]
This pull request serves for https://github.com/open-mmlab/mmaction2/issues/20.

**Comment by innerlee** [comment]
thanks!

File: configs/recognition/tsm/README.md [hunk 1]
```diff
@@ -4,24 +4,24 @@
...
+|config | resolution | gpus | backbone| pretrain | top1 acc| top5 acc | reference top1 acc | reference top5 acc | gpu_mem(M)  | ckpt | log| json|
+|:--|:--:|:--:|:--:|:--:|:--:|:--:|:--:|:--:|:--:|:--:|:--:|:--:|
+|[tsm_r50_1x1x8_50e_sthv1_rgb](/configs/recognition/tsm/tsm_r50_1x1x8_50e_sthv1_rgb.py) |short-side 100|8| ResNet50 | ImageNet|44.62|75.51|[42.08](https://github.com/mit-han-lab/temporal-shift-module/tree/8d53d6fda40bea2f1b37a6095279c4b454d672bd#training)|[72.66](https://github.com/mit-han-lab/temporal-shift-module/tree/8d53d6fda40bea2f1b37a6095279c4b454d672bd#training)| 7077| [ckpt](https://openmmlab.oss-accelerate.aliyuncs.com/mmaction/recognition/tsm/tsm_r50_1x1x8_50e_sthv1_rgb/tsm_r50_1x1x8_50e_sthv1_rgb_20200616-3417f361.pth) | [log](https://openmmlab.oss-accelerate.aliyuncs.com/mmaction/recognition/tsm/tsm_r50_1x1x8_50e_sthv1_rgb/20200616_022852.log)| [json](https://openmmlab.oss-accelerate.aliyuncs.com/mmaction/recognition/tsm/tsm_r50_1x1x8_50e_sthv1_rgb/20200616_022852.log.json)|
```
Comment (innerlee) [inline configs/recognition/tsm/README.md:17]: is sth-v1 height 100 or short-side 100?

File: configs/recognition/tsm/README.md [hunk 2]
```diff
@@ -4,24 +4,24 @@
...
+|config | resolution | gpus | backbone | pretrain| top1 acc| top5 acc |  reference top1 acc | reference top5 acc | gpu_mem(M)  | ckpt | log| json|
+|:--|:--:|:--:|:--:|:--:|:--:|:--:|:--:|:--:|:--:|:--:|:--:|:--:|
+|[tsm_r50_1x1x16_50e_sthv2_rgb](/configs/recognition/tsm/tsm_r50_1x1x16_50e_sthv2_rgb.py) |short-side 240|8| ResNet50| ImageNet |57.68 |83.65 |[56.57](https://github.com/mit-han-lab/temporal-shift-module/tree/8d53d6fda40bea2f1b37a6095279c4b454d672bd#training)|[84.30](https://github.com/mit-han-lab/temporal-shift-module/tree/8d53d6fda40bea2f1b37a6095279c4b454d672bd#training)| 10400| [ckpt](https://openmmlab.oss-accelerate.aliyuncs.com/mmaction/recognition/tsm/tsm_r50_1x1x16_50e_sthv2_rgb/tsm_r50_1x1x16_50e_sthv2_rgb_20200621-60ff441a.pth) | [log](https://openmmlab.oss-accelerate.aliyuncs.com/mmaction/recognition/tsm/tsm_r50_1x1x16_50e_sthv2_rgb/20200621_101921.log)| [json](https://openmmlab.oss-accelerate.aliyuncs.com/mmaction/recognition/tsm/tsm_r50_1x1x16_50e_sthv2_rgb/20200621_101921.log.json)|
```
Comment (innerlee) [inline configs/recognition/tsm/README.md:23]: is sth-v1 height 240 or short-side 240?

File: configs/recognition/tsm/README.md (hunk 1 above)
Comment (dreamerlin) [reply configs/recognition/tsm/README.md:17]: The short-side is the height. I think it's better to find the dataset creator to check the scripts @SuX97

File: configs/recognition/tsm/README.md (hunk 2 above)
Comment (dreamerlin) [reply configs/recognition/tsm/README.md:23]: > is sth-v2 height 240 or short-side 240?

The short-side is the height. I think it's better to find the dataset creator to check the scripts @SuX97

File: configs/recognition/tsm/README.md (hunk 1 above)
Comment (innerlee) [reply configs/recognition/tsm/README.md:17]: no. short-side is either width or height. for 9:16 videos, short-side is width

File: configs/recognition/tsm/README.md (hunk 2 above)
Comment (SuX97) [reply configs/recognition/tsm/README.md:23]: short-side does not specify whether it is height or width, as long as it is the shorter edge.

**Pull request merged**